flank-scripts/src/main/kotlin/flank/scripts/utils/Zip.kt

```diff
@@ -0,0 +1,73 @@
+package flank.scripts.utils
+
+import java.io.File
+import java.io.FileInputStream
+import java.util.zip.ZipEntry
+import java.util.zip.ZipFile
+import java.util.zip.ZipOutputStream
+
+fun unzip(src: File, dst: File = src.parentFile) {
+    print("* Unzipping: $src to $dst - ")
+    ZipFile(src).use { zipFile ->
+        zipFile.entries().asSequence().forEach { zipEntry ->
+            val outputFile = File(dst, zipEntry.name)
+
+            if (zipEntry.isDirectory) {
+                outputFile.mkdirs()
+                return@forEach
+            }
+
+            outputFile.parentFile.mkdirs()
+
+            zipFile.getInputStream(zipEntry).use { zipEntryInput ->
+                outputFile.outputStream().use { output ->
+                    zipEntryInput.copyTo(output)
+                }
+            }
+        }
```

Comment:
I would move it to separate function for better readability

Reply:
I am not really sure. This is a simple stupid utiliy function, not part of domain code.

Reply:
I agree with @piotradamczyk5

Reply:
2 vs 1. Ok, I am voted down